bin/morgue.js

```diff
@@ -255,6 +256,62 @@ function coronerError(argv, config) {
   throw Error(argv._[1]);
 }
 
+function coronerProject(argv, config) {
+  abortIfNotLoggedIn(config);
+
+  let subcommand = argv._[1];
+
+  var coroner = coronerClientArgv(config, argv);
+  var bpg = coronerBpgSetup(coroner, argv);
+
+  if(!subcommand) {
+    errx("Invalid project command. Try 'morgue project create <your-project-name>'")
+  }
+
+  if(subcommand === 'create') {
```

Comment:
small nit:
if you inverse the if statement - you can avoid putting all your code with additional tab.


```suggestion
  if(subcommand !== 'create') {
      errx("Invalid project command. Try 'morgue project create <your-project-name>'")
  }
  let project = argv._[2];
```

Reply:
I figured that down the line more subcommands may be added but that is better for the current function.